src/main/java/com/techswitch/ispy/controllers/SuspectsController.java

```diff
@@ -14,7 +14,7 @@
 import java.util.List;
 
 @RestController
-@CrossOrigin(origins = {"http://localhost:3000", "https://techswitch-i-spy-api-staging.herokuapp.com", "https://techswitch-i-spy-api.herokuapp.com"})
+@CrossOrigin(origins = {"http://localhost:3000", "https://techswitch-i-spy-staging.herokuapp.com/", "https://techswitch-i-spy.herokuapp.com"})
```

Comment:
Why does one have the `/` and the other 2 don't?

Reply:
...I remember fixing that....updated